src/components/popups/PopupGlossaryTermSelect.tsx

```diff
@@ -53,6 +54,10 @@ export const PopupGlossaryTermSelect = ({wide, codemirror}: { wide?: boolean, co
                 </InputGroup>
                 {isInlineTerm ?
                     <>
+                        <InputGroup className={"pl-4"}>
+                            <Label for={"glossary-term-titled-or-not"}>Titled glossary term?</Label>
+                            <Input type={"checkbox"} id="glossary-term-titled-or-not" onChange={e => setIsTitledTerm(e.target.checked)} checked={isTitledTerm}/>
```

Comment:
Unchecking "Inline glossary term?" should also `setIsTitledTerm(false)` to clean up any titled state.